tonic_validate/utils/llm_calls.py

```diff
@@ -391,18 +388,28 @@ async def statement_derived_from_context_call(
     return response_message
 
 
-def statement_derived_from_context_prompt(main_message):
+def statement_derived_from_context_prompt(statement: str, context_list: List[str]):
     """
 
     Parameters
     ----------
-    main_message : The main message to which additional instructions will be added.
+    statement: str
+        The statement to be checked.
+    context_list: List[str]
+        List of retrieved context.
 
     Returns
     -------
     prompt message for determining if a statement can be derived from context.
 
     """
+    if not context_list:
```

Comment:
Instead of having no context provided listed. Can we just put `EXAMPLE_CONTEXT` so it won't look out of place when someone tries to get the prompt from llama index without scoring?

Reply:
Yep, done.